lib/execution_engine2/sdk/SDKMethodRunner.py

```diff
@@ -462,7 +462,7 @@ def check_job_batch(
                 return_list=1,
             )["job_states"]
         return {
-            "parent_jobstate": parent_job_status,
+            "parent_jobstate": parent_job_status,  # change to batch_jobstate?
```

Comment:
👍 `batch` would be consistent with the nomenclature elsewhere

Reply:
done